Source/abci/Importer/aiMeshSchema.h

```diff
@@ -190,7 +211,12 @@ inline aiMeshSchema<T, U>::aiMeshSchema(aiObject * parent, const abcObject & abc
             // uv
             if (AbcGeom::IV2fGeomParam::matches(header))
             {
-                m_uv1_param = AbcGeom::IV2fGeomParam(geom_params, header.getName());
+                m_uv1_param = AbcGeom::IV2fGeomParam(geom_params, header.getName()); // obsolete
+            }
+
+            if (AbcGeom::IV2fGeomParam::matches(header))
```

Comment:
is this 2d Vector indexed properties.
We should probably include: Vector3 and Vector4 and Un-indexed flavours.